Assets/Scripts/Input/InputSystem.cs

```diff
@@ -8,6 +9,22 @@ public static class InputSystem
         private static InputActions _inputAction = new();
         private static Dictionary<System.Action<InputAction.CallbackContext>, InputAction> _subscribedInputs = new();
```

Comment:
Instead of putting `System` in front of the action you can make use of the `System` library at the top of the script (`using System;`)